aom/src/main/java/com/nedap/archie/adl14/ADL14DefaultMultiplicitiesSetter.java

```diff
@@ -0,0 +1,51 @@
+package com.nedap.archie.adl14;
+
+import com.nedap.archie.aom.Archetype;
+import com.nedap.archie.aom.CAttribute;
+import com.nedap.archie.aom.CObject;
+import com.nedap.archie.base.Cardinality;
+import com.nedap.archie.base.MultiplicityInterval;
+import com.nedap.archie.rminfo.MetaModel;
+import com.nedap.archie.rminfo.MetaModels;
+
+/**
+ * Sets the default occurrences, cardinality and existence with ADL 1.4 rules, if not explicitly set
+ * in a given Archetype. Useful for conversion to ADL 2, where the default values are different, and
+ * it is good to start with the correct values already present
+ */
+public class ADL14DefaultMultiplicitiesSetter {
+
+    private final MetaModels metaModels;
+
+    public ADL14DefaultMultiplicitiesSetter(MetaModels metaModels) {
+        this.metaModels = metaModels;
+    }
+
+    public void setDefaults(Archetype archetype) {
+        metaModels.selectModel(archetype);
+        correctItemsCardinality(archetype.getDefinition());
+    }
+
+    private void correctItemsCardinality(CObject cObject) {
```

Comment:
This doesn't actually change cardinalities. Changed method name to `correctItemsMultiplicities` for consistency.
```suggestion
        correctItemsMultiplicities(archetype.getDefinition());
    }

    private void correctItemsMultiplicities(CObject cObject) {
```